docs/how-to-integrate-a-frontend.md

```diff
@@ -0,0 +1,247 @@
+```
+
+Or directly clone the [repository](https://github.com/sCrypt-Inc/tic-tac-toe) that we have created. The `onlyweb` branch of this repository only contains the frontend code.
+
+```bash
+git clone -b onlyweb https://github.com/sCrypt-Inc/tic-tac-toe
+```
+![](../static/img/clone.png)
+
+Later, we will do most work under the `src` directory.
+
+### Install scryptTS SDK
+
+The sCrypt SDK - [scryptTS](https://scrypt.io/scrypt-ts) - enables you to easily compile, test, deploy, and call contracts.
+
+Step into the project directory, then use the `scrypt-cli` command line tool to install the SDK.
+
+```bash
+cd tic-tac-toe
+npx scrypt-cli init
+```
+
+This `init` command will also install the dependencies and initialize the contract development environment.
+
+After this initialization, we are ready to go!
+
+## Load Contract
+
+Before interacting with the contract in the front end, we need to load it first. 
+
+When a contract source file is compiled, it will deliver a JSON file, we call it the contract artifact.
+
+Generally, as a frontend developer, you will get this contract artifact from your backend partner or contract developer, then load it in the front end, and that's all.
+
+We'll take a look at how to generate the artifact by ourselves first.
+
+### Compile Contract
+
+When you have the contract source file, copy it into `src/contracts` directory.
+
+![](../static/img/copy-source-file.png)
+
+Then run the following command to compile the contract.
+
+```bash
+npx scrypt-cli compile
+```
+
+![](../static/img/compile.png)
+
+After the compilation, you will get the artifact in `artifacts/src/contract` directory.
+
+![](../static/img/artifacts.png)
+
+### Load Artifact
+
+Now with the artifact file `ticTacToe.json`, we directly load it in the `index.tsx` file.
+
+```ts
+import { TicTacToe } from './contracts/ticTacToe';
+var artifact = require('../artifacts/src/contracts/ticTacToe.json');
+TicTacToe.loadArtifact(artifact);
+```
+
+## Integrate Wallet
+
+We will integrate [Sensilet](https://sensilet.com/), a MetaMask-like wallet, into our project.
+
+### Signer and Provider
+
+When a user wants to send a transaction onto the chain, our dApp would prompt the user to “sign” the transaction using their private key and be authorized to perform certain actions. The `Signer` interface is an abstraction of the entity controlling the private key. A simple signer would be a single private key, while a complex signer is a wallet.
+
+A `Provider` is an abstraction for a Bitcoin node that you connect with when you need to interact with the blockchain, e.g., to broadcast a transaction. Whatsonchain is an example, providing access to the blockchain.
+
+### Connect Sensilet Wallet
+
+After clicking the **Connect Sensilet** button, we initialize a `SensiletSigner` and save it.
+
+In the `App.tsx` file, we then call the `getDefaultPubKey()` interface of the wallet to request to connect to get the public keys of the two players.
```

Comment:
`First we connect to Sensilet.`

Show how to connect using `getConnectedTarget`, not `getDefaultPubKey`.

Also update Learn section: `getConnectedTarget` should not be in test.